update-version.sh

```diff
@@ -26,6 +26,6 @@ function replace_in_poms() {
 }
 
 NEW_VERSION="$1"
-OLD_VERSION=$(grep "^    <version>" -h pom.xml | head -n1 | sed 's:[^/]*>\(.*\)</.*:\1:')
+OLD_VERSION=$(grep "^    <version>" -h pom.xml | sed -n '2p' | sed 's:[^/]*>\(.*\)</.*:\1:')
```

Comment:
The expression does not seem to retrieve anything (tried locally).

Can we use the `maven-help-plugin` to retrieve the project version, if that's the goal? It should be more reliable:
`mvn org.apache.maven.plugins:maven-help-plugin:3.2.0:evaluate -Dexpression=project.version -DforceStdout -q`

Reply:
sounds good - checking